cni/cmd/install/data/given/10-calico.conflist

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We usually use `testdata` and then `.golden` for expected files it would be good to keep this coherency